minari/utils/data_collector.py

```diff
@@ -0,0 +1,573 @@
+from __future__ import annotations
+
+import os
+import shutil
+import tempfile
+from typing import Any, Dict, Optional, Type, Union
+
+import gymnasium as gym
+import h5py
+import numpy as np
+from gymnasium import spaces
+
+
+STEP_DATA_KEYS = {
+    "actions",
+    "observations",
+    "rewards",
+    "truncations",
+    "terminations",
+}
```

Comment:
Why are we not keeping the info?

Reply:
I set recording infos as optional since some users may not find useful what is being returned. The argument is `record_infos` in `DataCollectorV0`, the default is `True` which does record the infos.